source/external-interface-design/external-interface-design.rst

```diff
@@ -155,21 +154,16 @@ This includes:
 
 -  IoT devices
 
--  Service profiles
-
 -  Device profiles
 
+-  Multicast groups.
+
 Changes to these entities must always happen in OS2iot, which is then
 synchronized to Chirpstack. It is not supported to change data directly
 in Chirpstack. All manipulation of settings in the Chirpstack is thus
-done via the Chirpstack API. If locally running on port 8080 then Chirpstack Application Services Swagger UI is available at http://localhost:8080/api
+done via the Chirpstack API. The protocol documentation for the api is documented at https://www.chirpstack.io/docs/chirpstack/api/api.html.
```

Comment:
Delete this sentence: `All manipulation of settings in the Chirpstack is thus
done via the Chirpstack API`